### ISSUE TITLE
Prevent bot to read and reply on any group messages when added to the group

### ISSUE THREAD

**Issue by tornadocontrib** [chofnar/release-bot#152]
The bot would reply on any messages from the group.

I think it should have a proper command keyword to prevent this.

**Comment by chofnar** [comment]
Thank you for bringing this to my attention. When I designed the bot, I did not think that it would be used in a group. I'll look into adjusting the logic soon.

**Comment by ghost** [comment]
@chofnar Yeah, I think this is an important feature. It works on group but the only problem is that the bot would reply to any group messages. I have setup the bot in case if I miss the release so that the other member could catch it
